src/style-flight-main.scss

```diff
@@ -43,25 +43,29 @@
   padding-bottom: var(--ds-size-25, $ds-size-25);
   color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
   font-size: var(--ds-text-heading-300-size, $ds-text-heading-300-size);
+  font-weight: 600;
```

Comment:
Auro doesn't support a `600` weight? Only `100`, `300` and `500`. See https://github.com/AlaskaAirlines/WebCoreStyleSheets/blob/master/src/_fonts.scss

Where is the `600` spec coming from?

Reply:
I'm not sure on this one, maybe @jordanjones243 could help answer to this?

Reply:
Font-weight 600 is coming as a request from the flight search team as they said the current UI does not match what they have in their design. Although, there is no blueprint for this component.

Reply:
@jordanjones243 is there a ticket for this font weight request? That needs to be done separate from this body of work.

@fajar-apri-alaska Since we don't have a font-weight 600 currently let's not include this part of the change and we will handle it in the ticket above.

Reply:
@leeejune is currently developing a blueprint for auro-flight. We can put this work on hold and remove it from this PR until that blueprint is finished.